src/Employee.java

```diff
@@ -1,3 +1,70 @@
 public class Employee {
+    private String oneName;
+    private String twoName;
+    private String threeName;
+    private int department;
+    private int salary;
+    private int ID;
```

Comment:
SCREAMING_CASE используется только для констант, id это не константа, а свойство класса. нужно использовать camelCase
```suggestion
    private int id;
```